src/utils/paginationHandler.js

```diff
@@ -1,130 +1,139 @@
+        }
+        if (pageIdentifier < 0) {
+            pageIdentifier =
+                paginator.maxNumberOfPages +
+                (pageIdentifier % paginator.maxNumberOfPages);
+        }
+        else if (pageIdentifier >= paginator.maxNumberOfPages) {
+            pageIdentifier %= paginator.maxNumberOfPages;
+        }
+        return Object.assign(Object.assign({}, paginator.currentIdentifiers), { pageIdentifier });
+    };
+    const buttonPaginator = new discord_js_pagination_1.ButtonPaginator(interaction, {
+        pages,
+        buttons,
+        identifiersResolver
+    });
+    buttonPaginator.on(discord_js_pagination_1.PaginatorEvents.PAGINATION_READY, (paginator) => {
+        for (const actionRow of paginator.messageActionRows) {
+            for (const button of actionRow.components) {
+                button.disabled = false;
+            }
+        }
+        return interaction.editReply(paginator.currentPage);
+    });
+    buttonPaginator.on(discord_js_pagination_1.PaginatorEvents.COLLECT_ERROR, () => __awaiter(void 0, void 0, void 0, function* () {
+        try {
+            /**
+             * @type {Message}
+             */
+            const message = yield interaction.fetchReply();
+            /**
+             * @type {EmbedBuilder}
+             */
+            const embed = message.embeds[0];
+            const components = message.components;
+            components[0].components.forEach((component) => {
+                component.disabled = true;
+            });
+            embed.setColor(config_1.config.COLORS.ERROR);
+            return interaction.editReply({
+                embeds: [embed],
+                components: components
+            });
+        }
+        catch (error) {
+            if (error.code)
+                return;
+            return console.log(error);
+        }
+    }));
+    buttonPaginator.on(discord_js_pagination_1.PaginatorEvents.PAGINATION_END, () => __awaiter(void 0, void 0, void 0, function* () {
+        try {
+            /**
+             * @type {Message}
+             */
+            const message = yield interaction.fetchReply();
+            /**
+             * @type {EmbedBuilder}
+             */
+            const embed = message.embeds[0];
+            const components = message.components;
+            components[0].components.forEach((component) => {
+                component.disabled = true;
+            });
+            embed.setColor(config_1.config.COLORS.ERROR);
+            return interaction.editReply({
+                embeds: [embed],
+                components: components
+            });
+        }
+        catch (error) {
+            if (error.code)
+                return;
+            return console.log(error);
+        }
+    }));
+    yield buttonPaginator.send();
+    return buttonPaginator.message;
+});
+exports.paginationHandler = paginationHandler;
+>>>>>>> Stashed changes
```

Comment:
See above

Reply:
@Juilzze resolved this issue in latest commit.